source/common/network/connection_impl_base.cc

```diff
@@ -42,10 +40,6 @@ void ConnectionImplBase::setDelayedCloseTimeout(std::chrono::milliseconds timeou
   delayed_close_timeout_ = timeout;
 }
 
-ExecutionContext* ConnectionImplBase::executionContext() const {
```

Comment:
It seems this class (`ConnectionImplBase`) is the base of TCP and QUIC connections.   Why not just change this function to `ConnectionImplBase::trackedStream`, such that both kinds of connections are covered?

Reply:
Because dumpState is implement in the ConmectionImpl, I hope these two methods could be implemented at same place.

Reply:
In that case, can you add a `trackedStream()` override in `source/common/quic/quic_filter_manager_connection_impl.h` to cover QUIC connections?

Reply:
okay.

Reply:
moved the trackedStream() to the ConnectionImplBase to avoid repeat it in the client session and server seesion of quic.